poppy/optics.py

```diff
@@ -347,6 +347,20 @@ def get_transmission(self, wave):
         return res
 
 
+class ScalarOpticalPathDifference(AnalyticOpticalElement):
+    """Uniform and constant optical path difference
+
+    """
+    @utils.quantity_input(opd=u.meter)
+    def __init__(self, name=None, opd=1.0*u.micron, **kwargs):
+        AnalyticOpticalElement.__init__(self, name=name, **kwargs)
+        self.opd = opd
+
+    def get_opd(self, wave):
+        res = np.empty(wave.shape, dtype=_float())
```

Comment:
Did you mean `dtype=float` here? The same `_float()` call is also present on line 1892, but it doesn't seem to work when I try it locally with `np.empty(a.shape, dtype=_float())` where `a` is an array.

Reply:
no, the `_float()` with leading underscore is intentional. That's a little utility function defined in `accel_math.py`:
```
def _float():
    """ Returns numpy data type for desired precision based on configuration """
    # How many bits per float to use?
    return np.float64 if conf.double_precision else np.float32
```